frontend/src/js/api/api.ts

```diff
@@ -398,7 +399,7 @@ export const useGetEntityHistory = () => {
         max: string; // Format like "2020-12-31"
       } = {
         min: "2015-01-01",
```

Comment:
you don't have to send a min, is probably safer than wondering why entries are missing

Reply:
Is `max` optional as well? Is the entire `time` object optional?